changelogs/unreleased/fix-netbox-index-schema-path.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/core
```

Comment:
Re commit message: I hope this change may be described in 50 symbols.

Reply:
I did the same as in https://github.com/tarantool/tarantool/commit/84a2f70794bc0672599109f50a8caa66436b643d

Updated.

Reply:
Personally I dislike issue / commit numbers in commit titles (a subject should be descriptive), but I don't want to bother you with that.